pkg/buildpack/buildpack.go

```diff
@@ -100,31 +137,36 @@ func (b *distBlob) Open() (io.ReadCloser, error) {
 	return b.openFn(), nil
 }
 
-func toDistTar(tw archive.TarWriter, bpd dist.BuildpackDescriptor, blob Blob) error {
+func toDistTar(tw archive.TarWriter, descriptor Descriptor, blob Blob) error {
 	ts := archive.NormalizedDateTime
 
+	parentDir := dist.BuildpacksDir
+	if descriptor.Kind() == "extension" {
```

Comment:
Should we make `"extension"` a const somewhere? I see it used a few times

Reply:
Updated in [a257564](https://github.com/buildpacks/pack/pull/1478/commits/a257564cf8b45a62ca93ce90e011025d9c4c05c8)